### DISCUSSION TITLE
Changing /_next/data domain

### DISCUSSION THREAD

**Discussion by tnovau** [vercel/next.js#28490] [Help]
Hi, we have a webapp that is loaded on a path of our domain:

`https://www.example.com/this-path`

We're using `getServerSideProps` to fetch data on SSR. And we're facing some issues with `/_next/data` requests because we can't specify to Next.js what is the origin to fetch that data.

By default it's trying to fetch it on `https://www.example.com/_next/data/[...]` and we can't add the rule to our balancer because we have some Next.js apps out there.

Is there a way to change the origin from `https://www.example.com` to `https://www.this-link.com`?

**Comment by yuuk** [comment]
What I want is to change _next/data prefix, like `assetPrefix`
